source/_integrations/zone.markdown

```diff
@@ -10,17 +10,51 @@ ha_codeowners:
   - '@home-assistant/core'
 ha_domain: zone
 ha_integration_type: system
+related:
+  - docs: /docs/configuration/basic/
+    title: Editing basic settings
+  - docs: /docs/organizing/areas/
+    title: Areas
+  - docs: /getting-started/onboarding/
+    title: Onboarding
 ---
 
-Zones allow you to specify certain regions on earth (for now). When a device tracker sees a device to be within a zone, the state will take the name from the zone. Zones can also be used as a [trigger](/docs/automation/trigger#zone-trigger) or [condition](/docs/scripts/conditions/#zone-condition) inside automation setups.
+Zones allow you to specify certain regions on a map. When a device tracker sees a device to be within a zone, the state will take the name from the zone. Zones can also be used as a [trigger](/docs/automation/trigger#zone-trigger) or [condition](/docs/scripts/conditions/#zone-condition) inside automations. For example, for showing the weather, opening the shades at sunrise, or starting the vacuum when you leave the home.
 
-Zones can be added and managed through the user interface at {% my zones title="**Settings** > **Areas, labels & zones**" %}.
+## About the home zone
 
-![Screenshot of the UI for adding or editing a zone](/images/integrations/zone/zone_edit_ui.png)
+During the [onboarding process](/getting-started/onboarding/), Home Assistant asked for your home location. You either entered this manually or asked Home Assistant to detect it automatically.
```

Comment:
IMHO, this should be combined with the next paragraph. They talk about the same location, and it makes reading/"connecting the dots" easier.

```suggestion
During the [onboarding process](/getting-started/onboarding/), Home Assistant asked for your home location. You either entered this manually or asked Home Assistant to detect it automatically. This location was used to create the home zone with a 100&nbsp;m radius. The home zone is a special, pre-defined zone with a few characteristics that set it apart from other zones.
```